ANSWERS.md

```diff
@@ -1,6 +1,12 @@
 ### Questions:
-1. What is the DOM?
+1. What is the DOM? 
+    Document Object Model
```

Comment:
Let's try to explain a little more that way I know you understand more than just the abbreviation ;)